### PR TITLE
fix #116871: add support for rt_decay

### PR DESCRIPTION
**Pull request by hpfmn**

### CODE REVIEW

File: zerberus/voice.cpp [hunk 1]
```diff
@@ -120,8 +121,12 @@ void Voice::start(Channel* c, int key, int v, const Zone* z)
       if (trigger == Trigger::CC)
             _velocity = 127;
       float curve = _velocity * _velocity / (127.0 * 127.0);
+
+      double rt_decay_value = 1.0;
+      if (trigger == Trigger::RELEASE)
+            rt_decay_value = pow(10, (-z->rtDecay * durSinceNoteOn)/20);
```
Comment (lasconic) [inline zerberus/voice.cpp]: There is too much magic for me on this two lines :)

File: zerberus/voice.h [hunk 2]
```diff
@@ -128,6 +128,7 @@ class Voice {
       int _loopStart;
       int _loopEnd;
       bool _looping;
+      int _ticks; // living duration in samples
```
Comment (lasconic) [inline zerberus/voice.h]: ticks is already used in musescore with another meaning. Any better name?

File: zerberus/voice.cpp (hunk 1 above)
Comment (hpfmn) [inline zerberus/voice.cpp]: 0.005 was there from the beginning - not sure where this comes from - I think just something to have a little more headroom. Maybe ask Werner if he knows something? The 20 comes from decibel https://en.wikipedia.org/wiki/Decibel#Field_quantities_and_root-power_quantities

File: zerberus/voice.h (hunk 2 above)
Comment (hpfmn) [inline zerberus/voice.h]: Maybe _samplesSinceStart?

Comment (lasconic) [inline zerberus/voice.h]: better indeed.

File: zerberus/voice.cpp [hunk 3]
```diff
@@ -120,8 +121,12 @@ void Voice::start(Channel* c, int key, int v, const Zone* z)
...
       gain        = z->volume * (offset + z->ampVeltrack * curve)
-                    * .005 * c->gain();
+                    * .005 * c->gain() * rt_decay_value;
```
Comment (lasconic) [inline zerberus/voice.cpp]: @wschweer Any idea where this 0.005 comes from?

**Pull request merged**